app/views/content/assessment-only-providers.md

```diff
@@ -502,14 +502,14 @@ keywords:
   - experienced
 ---
 
-If you’re an experienced teacher with a degree, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
+If you have a degree and enough classroom experience, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
 
 The accredited teacher training providers listed on this page offer the AO programme.
 
 Contact them directly to ask them about entry criteria and to apply.
 
 ### Teachers from outside the UK
 
-If you trained as a teacher outside the UK, you can apply to certain training providers for an assessment only QTS without visiting or training in England. 
+If you have experience of teaching outside the UK, you can apply to certain training providers for an assessment only QTS without visiting or training in England. 
```

Comment:
Is this definitely right @tcannon3 - that you only need experience, not training, as an international AO applicant?

(That might well be the case - just wanted to check!)

Reply:
I read on TES that international AO applicants can be any member of school staff -unqualified teacher, TA or higher-level teaching assistant with significant teaching experience. Basically it's someone who can prove that they meet all of the Teaching Standards after having significant classroom experience in a British Overseas school

Reply:
Having said that, I've just double checked their entry requirements and it seems that they want to international applicant to be employed in a teaching role within a school. But that doesn't necessarily mean trained teacher as the candidate could be an unqualified teacher

Reply:
@gemmadallmandfe Phoebe has given the OK for these changes